test/fixtures/dialects/exasol/CreateAdapterScriptStatement.yml

```diff
@@ -3,70 +3,5 @@
 # computed by SQLFluff when running the tests. Please run
 # `python test/generate_parse_fixture_yml.py`  to generate them after adding or
 # altering SQL files.
-_hash: f0cd1f084b6632614727d428c6e6940b0d8353cb9d9f2b3975f365ff4da8ab7b
-file:
-- statement:
-    create_adapter_script:
-    - keyword: CREATE
-    - keyword: JAVA
-    - keyword: ADAPTER
-    - keyword: SCRIPT
-    - script_reference:
-        identifier: my_script
-    - keyword: AS
-    - script_content:
-      - raw: '%'
-      - raw: jar
-      - raw: hive_jdbc_adapter
-      - raw: .
-      - raw: jar
-      - raw: ;
-- function_script_terminator: /
-- statement:
-    create_adapter_script:
-    - keyword: CREATE
-    - binary_operator: OR
-    - keyword: REPLACE
-    - keyword: PYTHON
-    - keyword: ADAPTER
-    - keyword: SCRIPT
-    - script_reference:
-      - identifier: test
-      - dot: .
-      - identifier: adapter_dummy
-    - keyword: AS
-    - script_content:
-      - raw: def
-      - raw: adapter_call
-      - bracketed:
-          start_bracket: (
-          raw: in_json
-          end_bracket: )
-      - raw: ':'
-      - raw: return
-      - raw: '"BLABLA"'
-- function_script_terminator: /
-- statement:
-    create_adapter_script:
-    - keyword: CREATE
-    - binary_operator: OR
-    - keyword: REPLACE
-    - keyword: LUA
-    - keyword: ADAPTER
-    - keyword: SCRIPT
-    - script_reference:
-      - identifier: test
-      - dot: .
-      - identifier: adapter_dummy
-    - keyword: AS
-    - script_content:
-      - raw: function
-      - raw: adapter_call
-      - bracketed:
-          start_bracket: (
-          raw: in_json
-          end_bracket: )
-      - raw: ':'
-      - raw: return
-      - raw: "'BLABLA'"
-- function_script_terminator: /
+_hash: fdda373cd9cd649f82a9c5cf7ba9e290375c0ceae29477b0bad5a25f24a52ae3
+file: null
```

Comment:
What's happening here?

Reply:
Essentially, if you remove greedy delimiters, there is no way to support two separate terminator types based on our current tooling for dialects. This is something I intend to remedy by making some more complex types, but I don't want to do too much in one PR. This is basically the only trade off with adding this PR in - we cannot support Exasol functions. They have neither the normal delimiter, nor any symbol which indicates the start and end of the 'anything goes' bit. We can't have two delimiters, as we can't do terminator inheritance, or conditionally matching based on matched children yet. So, for now, I've elected to treat them like we do comments and psql meta commands. Their presence won't fail anything, but we can't lint them. Shouldn't have a big impact really as most of the function is the 'anything goes' bit that we can't lint anyway.